sns_ik_lib/src/sns_ik_math_utils.cpp

```diff
@@ -22,18 +22,19 @@
 
 #include <sns_ik/sns_ik_math_utils.hpp>
 
-using namespace Eigen;
-using namespace sns_ik;
+namespace {
```

Comment:
Why put this one constant in an anonymous namespace?

Reply:
My goal was to move the constants to the most restricted namespace that was possible. This constant was only needed in the source file, so I moved it there and prevented it from ever being exported by using the anonymous namespace. Probably overkill. On a future pass I will go through and document all of the constants that are scattered throughout the code.

There are several hard-coded constants throughout the project that I plan to document and make more clear at a later date.

I also put the other two constants in this class into an anonymous namespace, but it was in the header file because they were used there.